src/scope.ml

```diff
@@ -401,9 +416,15 @@ let scope_rule : sig_state -> p_rule -> sym * pp_hint * rule loc = fun ss r ->
     let (h, args) = Basics.get_args lhs in
     let is_const s = s.sym_mode = Const in
```

Comment:
This function should be moved to Terms (or Basics) where there is already is_inj.

Reply:
Should a symbol with no rewrite rules be considered constant, even if it hasn't been marked `const`?

Reply:
No.